src/auto.ts

```diff
@@ -227,11 +227,15 @@ export default class Auto {
     const target_url = url;
 
     if (!dryRun) {
-      await this.git.createStatus({
-        ...options,
-        sha,
-        target_url
-      });
+      try {
+        await this.git.createStatus({
+          ...options,
+          sha,
+          target_url
+        });
+      } catch {
+        this.logger.log.error('Failed to post status to Pull Request.');
```

Comment:
If `this.git.createStatus` throws an exception, wouldn't we print: `'Failed to post status to Pull Request.'` and `'Posted status to Pull Request.'` right after it? Do we want to swallow this error?